debs/SPECS/3.10.0/wazuh-manager/debian/postrm

```diff
@@ -72,6 +59,18 @@ case "$1" in
     ;;
 
     purge)
+        if getent passwd | grep -q "^ossecr" ; then
```

Comment:
This change is ok but the indentation in some lines are tabs and in other are spaces. Can you change them to spaces?